src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationNotifications.cs

```diff
@@ -23,7 +23,6 @@ public OpenIdConnectAuthenticationNotifications()
             MessageReceived = notification => Task.FromResult(0);
             SecurityTokenReceived = notification => Task.FromResult(0);
             SecurityTokenValidated = notification => Task.FromResult(0);
-            RedirectToIdentityProvider = notification => Task.FromResult(0);
```

Comment:
We should sit down and review the whole notification design before you apply this to any additional events. We want to standardize the notification pattern across all the auth handlers.